src/applications/mhv/medications/components/PrescriptionDetails/VaPrescription.jsx

```diff
@@ -153,10 +153,22 @@ const VaPrescription = prescription => {
                         height="350"
                       />
                     ) : (
-                      <p>(Image not available)</p>
+                      <p>Image not available</p>
                     )}
                   </va-additional-info>
                 </div>
+                <div className="print-only">
+                  {entry.cmopNdcNumber ? (
+                    <img
+                      src={getImageUri(entry.cmopNdcNumber)}
+                      alt="Not Available"
+                      width="350"
+                      height="350"
+                    />
+                  ) : (
+                    <p className="vads-u-margin--0">Image not available</p>
```

Comment:
I think this may be a duplicate from line 149?

Reply:
Gotcha, yeah i missed that print-only class on my initial review. So then that makes me curious now as to why the alt is different on 151 from the print-only view?